package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "vscode-docker",
-    "version": "1.13.0",
+    "version": "1.13.1-alpha",
```

Comment:
Should this be 1.13.5-alpha?

Reply:
`1.13.1-alpha` is fine.  It just needs to be at least `0.0.1` more then the previous version.